presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/OptimizeMixedDistinctAggregations.java

```diff
@@ -122,6 +122,11 @@ public PlanNode visitAggregation(AggregationNode node, RewriteContext<Optional<A
                 return context.defaultRewrite(node, Optional.empty());
             }
 
+            if (!node.getOrderBySymbols().isEmpty()) {
```

Comment:
this should be probably part of `Augment AggregationNode with ordering information` commit